### PR TITLE
Items index style

### PR DESCRIPTION
**Pull request by vaidashi**
LIST OF CHANGES:
-paginated items index page, formatted item show page to decrease white space
-slightly formatted root page, style of font and size of fonts 
-fixed footer to bottom of page, centered text, added margins
-

THINGS I'M UNSURE ON:
- weird box around selected items on root, we can keep or address
-
-

RSPEC GREEN?
**YES** 
NO- WHY?

*MENTION GROUP MEMBER TO REVIEW PR*
@jimszalew @mimilettd

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -46,10 +46,11 @@
 .loginBtn--twitter:focus {
   background: #66ccff;
 }
+
 .item {
   height: 650px;
   border-style: solid;
-  border-width: 2px;
+  border-width: 1px;
   border-radius: 5px;
```
Comment (mimizotti) [inline app/assets/stylesheets/application.scss:54]: Thanks for making this minor change!

File: app/assets/stylesheets/fonts.css [hunk 2]
```diff
@@ -16,3 +16,26 @@ p.logo {
...
+div.degination {
+  text-align: center;
+  margin-top: 15%;
+  font-family: 'Lora', serif;
+}
+
+p.card-text {
+  text-align: center;
+  font-weight: bold;
+}
```
Comment (mimizotti) [inline app/assets/stylesheets/fonts.css:41]: Nice use of classes here.

**Review by mimizotti — APPROVED** [review]
@vaidashi you did some great work here. thanks for implementing the pagination on items. Will you be able to do the same for orders living in the `Admin Dashboard` views? Thanks!

**Comment by jimszalew** [comment]
Legit

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (liambarstad) [reply app/assets/stylesheets/application.scss:54]: thank you!

**Pull request merged**